src/app/app.component.html

```diff
@@ -35,13 +53,18 @@
             </li>
         </ul>
     </div>
+
+    <!-- if login done then show this  -->
+    <div class="bg-dark p-4" *ngIf="router.url == '/dashboard' || router.url == '/group'">
+
+        <ul class="navbar-nav text-white">
+            <li class="nav-item"><a class="dropdown-item" href="#">Profile</a></li>
+            <li class="nav-item"><a class="dropdown-item" routerLink="/">Logout</a></li>
+        </ul>
```

Comment:
For profile & logout option, create an array in component and use *ngFor in html

Reply:
Issue still not resolved, use *ngFor with dropdownMenus